CDAlertView/Classes/CDAlertView.swift

```diff
@@ -399,10 +406,20 @@ open class CDAlertView: UIView {
                     self.popupView.center = self.backgroundView.center
                 })
             }
-        } else if recognizer.state == .cancelled {
+            break
+        case .cancelled:
             UIView.animate(withDuration: 0, animations: {
                 self.popupView.center = self.backgroundView.center
             })
+            break
+        case .began:
+            break
+        case .changed:
+            break
+        case .failed:
+            break
+        case .possible:
+            break
```

Comment:
we can just write 
```
default:
  break
```
instead of using all cases. So, removing `began`, `changed`, `failed`, `possible` and putting `default` instead looks cleaner for me.

Reply:
Okay! ƪ(•ᴗ•ƪ)